modelerfour/test/scenarios/a-playground/flattened.yaml

```diff
@@ -1675,7 +1675,7 @@ schemas: !<!Schemas>
           default:
             name: MyDerivedType
             description: ''
-            namespace: Api20160229
```

Comment:
So should we expect to see all namespaces go away until multiple API versions are detected?

Reply:
They can pass `--use-model-namespace:false` to turn them back on. It default to the logic in #173 to figure out what to do.